certificate-service/app/src/test/java/se/inera/intyg/certificateservice/application/citizen/service/PrintCitizenCertificateServiceTest.java

```diff
@@ -55,6 +56,21 @@ void shouldReturnPrintCitizenCertificateResponse() {
             ADDITIONAL_INFO_TEXT
         )
     ).thenReturn(PDF);
+  }
+
+  @Test
+  void shouldValidateCertificateId() {
```

Comment:
We need test for validate person id also

Reply:
Sorry my mistake just rename the test since you're validating both id and person id :)